.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -65,6 +57,7 @@ body:
     attributes:
       label: Pages Deployment Method
       options:
+        - N/A
```

Comment:
developers could have not yet deployed their app (and seeing an error during dev building)